js/index.js

```diff
@@ -1,23 +1,35 @@
 // ITERATION 1
 
 function updateSubtotal(product) {
-  console.log('Calculating subtotal, yey!');
-
-  //... your code goes here
+  const price = product.querySelector('.price span');
+  const quantity = product.querySelector('.quantity input');
+  const priceValue = price.innerHTML;
+  const quantityValue = quantity.value;
+  const subTotalPrice = priceValue * quantityValue;
+  const subTotal = product.querySelector('.subtotal span');
+  const subTotalCalc = (subTotal.innerHTML = subTotalPrice);
```

Comment:
If you will not be using this variable again, do not need to declare it. You can simply write the functionality -> find the element and update the innerHTML of it.

```
product.querySelector('.subtotal span').innerHTML = subTotalPrice;
```